.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,66 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug", "triage"]
+assignees:
+  - octocat
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: input
+    id: contact
+    attributes:
+      label: Contact Details
+      description: How can we get in touch with you if we need more info?
+      placeholder: Please visit out slack chanel [slack](https://adminbro.slack.com/archives/C011WETJSCB) or leave your email.
+    validations:
+      required: false
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Also tell us, what did you expect to happen?
+      placeholder: Tell us what you see!
+      value: "A bug happened!"
+    validations:
+      required: true
+  - type: input
+    id: prevalence
+    attributes:
+      label: Bug prevalence
+      description: "How often do you or others encounter this bug?"
+      placeholder: "Example: Whenever I visit the personal account page (1-2 times a week)"
+    validations:
+      required: true
+  - type: input
+    id: version
+    attributes:
+      label: AdminJS version
+      description: "on which version you encounter this bug?"
+      placeholder: "copy exact version that u are currently using"
+    validations:
+      required: true
+  - type: dropdown
+    id: browsers
+    attributes:
+      label: What browsers are you seeing the problem on?
+      multiple: true
+      options:
+        - Firefox
+        - Chrome
+        - Safari
+        - Microsoft Edge
+  - type: textarea
+    id: logs
+    attributes:
+      label: Relevant log output
+      description: Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
+      render: bash
+  - type: textarea
+    id: code
+    attributes:
+      label: Relevant code that giving u issues
+      description: Please copy and paste any relevant code. This will be automatically formatted into code, so no need for backticks.
+      render: TypeScript
```

Comment:
Do we have a place for e. g. `Additional info` where people can add screenshots etc?

Reply:
I think you can't attach a file

Reply:
sorry, you can but it is just by drag&drop